api/src/pcapi/core/finance/api.py

```diff
@@ -705,7 +708,10 @@ def _payment_details_row_formatter(sql_row):
     if sql_row.educational_booking_id is not None:
         booking_type = "EACC"
     elif sql_row.individual_booking_id is not None:
-        booking_type = "PC"
+        if sql_row.deposit_type == payments_models.DepositType.GRANT_15_17:
+            booking_type = "EACI"
+        elif sql_row.deposit_type == payments_models.DepositType.GRANT_18:
+            booking_type = "PC"
```

Comment:
Si on ne pass ni dans le if ni dans le elif que tu as ajoutés, la fonction renvoie `None`. J'aurais plutôt envie de lever une erreur, parce que c'est louche. Une adaptation comme suit pourrait faire le boulot : 

```suggestion
    if sql_row.educational_booking_id is not None:
        booking_type = "EACC"
    elif sql_row.deposit_type == payments_models.DepositType.GRANT_15_17:
        booking_type = "EACI"
    elif sql_row.deposit_type == payments_models.DepositType.GRANT_18:
        booking_type = "PC"
    else:
        raise ValueError(...)
```